rewrite-core/src/main/java/org/openrewrite/shell/exec/NpmExecutor.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2024 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.shell.exec;
+
+import java.nio.file.Path;
+
+@SuppressWarnings("unused")
+public class NpmExecutor implements ShellExecutor {
+    public void init(Path dir) {
```

Comment:
Same as below for `postExec`, this is not quite an override, but feels like it maybe should be, depending on what you were after.